content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +198,52 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
 
 Channel promotion is the notion of moving an operator from one channel to 
 another.  For example, consider the case where you have an operator version 
-1.0.1 which is found in a candidate channel, then you might decide to offer 
+`1.0.1` which is found in a candidate channel, then you might decide to offer 
 support for that version and want to move it to a stable channel.  
 
 Today, channel promotion is achieved by creating a new operator version 
-(1.0.2) that is labeled with the channel(s) you want to promote to (as well 
+(`1.0.2`) that is labeled with the channel(s) you want to promote to (as well 
 as any channels you want to keep it in).
+
+# UPGRADES 
+
+OLM provides a variety of ways to specify updates between operator versions. Before
+continue in the following recommendations, make sure you understand its options by checking [Creating an update graph with OLM][upgrade-graph].
+
+## Recommended upgrade path for patch release (bug fixes)
+
+Within a channel each patch release should be directly upgradeable. Use skip or skipRange
+to ensure its behaviour. 
+
+#### Example
+
+If you have the channel `stable-3.7` where the head of channel is `3.7.10` and then, you will provide a new patch release
+with a bug fix using the Operator bundle version `3.7.11` then, ensure that you configure your new publication to skip all 
+that was published in under the channel `stable-3.7` (e.g. skipRange: `">= 3.7.0 < 3.7.11"` replaces: `3.7.10` ).  
+
+So that, you can ensure that your users can move forward easier to the latest version since in this range `">= 3.7.0 < 3.7.11"`
+you just provided bug fixes and not new features or breaking changes.
+
+## Recommended upgrade path cross-channel
+
+After to be subscribed to a channel, OLM users are able to update their [Subscription][subscription] to
+consume the solutions from another channel, for example, by moving from the fast channel to
+the stable one.
+
+Because of this, it is recommended configure your bundles versions to allow cross-channel updates at least 
+between two consecutive major/minor versions (e.g. by having installed the head of the channel `stable-2.9` 
+your users can get installed the first version under `stable-2.10`)
+
+#### Example
+
+If you have published solutions on the channel `stable-3.5`
+then, you would like to begin to distribute Operator bundle versions `3.6.x`
+that would mean distributed a new bundle under `stable-3.6` skipping
+all versions which are provided via the `stable-3.5` channel. 
+
+So that, you can ensure that your users can move forward from any version that they have installed promoted via the 
+`stable-3.5` channel to the first version of the channel `stable-3.6`. (e.g. `skipRange: ">=3.5.0 <3.6.0"`)
```

Comment:
Caveat to call out here is this means that the operator author must be comfortable that any 3.6.z version can upgrade from any 3.5.z version.